relayer/relays/beefy/scanner.go

```diff
@@ -19,7 +20,7 @@ type ScanBlocksResult struct {
 
 func ScanBlocks(ctx context.Context, meta *types.Metadata, api *gsrpc.SubstrateAPI, startBlock uint64) (chan ScanBlocksResult, error) {
 	results := make(chan ScanBlocksResult)
-	go scanBlocks(ctx, meta, api, startBlock, results)
+	go scanBlocks(ctx, meta, api, 25420890, results)
```

Comment:
Hard code block number to 5 blocks before the block that has the bad commitment so that we can skip directly to the block we want.